### PR TITLE
Migrate to new sequence model and decoders in lucuma-odb library

### PR DESCRIPTION
**Pull request by rpiaggio**

### CODE REVIEW

**Comment by rpiaggio** [comment]
I recommend reviewing with the "Hide whitespace" option.

File: build.sbt [hunk 1]
```diff
@@ -1,11 +1,12 @@
 val disciplineMUnitVersion = "1.0.9"
-val lucumaCoreVersion      = "0.76.0"
+val lucumaCoreVersion      = "0.80.0"
+val lucumaODBSchema        = "0.3.0"
```
Comment (cquiroz) [inline build.sbt:3]: Are we getting decoders from the odb now?

Comment (rpiaggio) [reply build.sbt:3]: Yes

File: lucuma-schemas/src/test/scala/lucuma/schemas/decoders/InputStreamSuite.scala [hunk 2]
```diff
@@ -30,6 +30,8 @@ trait InputStreamSuite extends CatsEffectSuite {
   def assertParsedStreamEquals[A: Decoder](jsonFile: String, expected: A): IO[Unit] =
     for {
       r <- parsedResult[A](jsonFile)
+      // _ <- IO.println("HELLO!!!")
```
Comment (cquiroz) [inline lucuma-schemas/src/test/scala/lucuma/schemas/decoders/InputStreamSuite.scala]: Commented code

File: lucuma-schemas/src/test/scala/lucuma/schemas/decoders/ManualSequenceDecodersSuite.scala [hunk 3]
```diff
@@ -1,105 +1,105 @@
...
-import lucuma.core.enums.*
-import lucuma.core.math.Offset
-import lucuma.core.math.Wavelength
-import lucuma.core.model.sequence.*
-import lucuma.core.syntax.timespan._
-import lucuma.refined.*
+// import cats.syntax.all.*
+// import eu.timepit.refined.numeric.Positive
+// import lucuma.core.enums.*
+// import lucuma.core.math.Offset
```
Comment (cquiroz) [inline lucuma-schemas/src/test/scala/lucuma/schemas/decoders/ManualSequenceDecodersSuite.scala]: Commented code

File: model/src/main/scala/lucuma/schemas/decoders/CoreModelDecoders.scala [hunk 4]
```diff
@@ -108,21 +105,21 @@ trait CoreModelDecoders {
 
   given Decoder[ProperMotion] = semiauto.deriveDecoder[ProperMotion]
 
-  given Decoder[Duration] = Decoder.instance(
-    _.downField("microseconds")
-      .as[Long]
-      .map(l => Duration.of(l, ChronoUnit.MICROS))
-  )
+  // given Decoder[Duration] = Decoder.instance(
```
Comment (cquiroz) [inline model/src/main/scala/lucuma/schemas/decoders/CoreModelDecoders.scala]: Are these now in the odb?

**Review by cquiroz — APPROVED** [review]
LGTM

File: model/src/main/scala/lucuma/schemas/decoders/CoreModelDecoders.scala (hunk 4 above)
Comment (rpiaggio) [reply model/src/main/scala/lucuma/schemas/decoders/CoreModelDecoders.scala]: Yup

**Pull request merged**